Gopkg.toml

```diff
@@ -169,3 +169,15 @@ required = [
 [[override]]
   name="github.com/golang/protobuf"
   version = "1.3.2"
+
+[[override]]
+  name = "github.com/golang/glog"
+  # Redirect glog calls -> zap (if dependencies end up using it)
+  branch = "enable-levels"
+  source = "https://github.com/coryrc/glog.git"
+
+[[override]]
+  name = "k8s.io/klog"
+  # Redirect klog calls -> zap
+  branch = "enable-levels"
+  source = "https://github.com/coryrc/klog.git"
```

Comment:
Do we want to depend on personal forks for these? Any chance to upstream the relevant changes?

Reply:
Waiting on knative/community#76 (and maybe knative/community#69). I can wait if need be (though need knative/pkg#1118 in).